app/views/layouts/_seo.html.erb

```diff
@@ -0,0 +1,50 @@
+<%# This belongs inside the <head> tag %>
+
+<link rel="publisher" href="https://plus.google.com/u/0/+IndentLabs" />
```

Comment:
I thought so, thanks for clearing that up. I'll use the link that you put on the main page.